script/main.py

```diff
@@ -4,15 +4,25 @@
 
 
 def fetch_argument():
-  parser = argparse.ArgumentParser()
+  parser = argparse.ArgumentParser()   
+  parser.add_argument('--driver', help='entry driver')
   parser.add_argument("url", help="entry page url")
-  args = parser.parse_args()
-  return(args.url)
+  args = parser.parse_args()     
+  return(args.driver, args.url)
```

Comment:
BTW you don't need parentheses here. Technically it is correct but most of the time people don't do it in python.
Also add space after return.